microsoft-edge/devtools-guide-chromium/performance/reference.md

```diff
@@ -0,0 +1,1238 @@
+* A lighter-blue square means lower-priority.
+
+For example, in the previous figure, the **photo-gallery** request, in the top left corner of the **Network** section is higher-priority.
+
+Requests might have lines on the left and right sides, and their bars might be split into two colors.  Here's what these lines and colors represent:
+
+* The left line is everything up to the `Connection Start` group of events, inclusive.  In other words, it's everything before `Request Sent`, exclusive.
+
+* The light portion of the bar is `Request Sent` and `Waiting (TTFB)` for server response.
+
+* The dark portion of the bar is `Content Download`.
+
+* The right line is essentially time spent waiting for the main thread.  This isn't represented in the **Timing** tab.
+
+
+<!-- ------------------------------ -->
+#### View memory metrics
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#memory -->
+
+Select the **Memory** checkbox to view memory metrics in a performance recording:
+
+![The Memory checkbox](./reference-images/memory-highlight.png)
+
+DevTools displays a **Memory** chart, above the **Summary** tab.  There is also a chart below the **NET** chart, called **HEAP**.  The **HEAP** chart provides the same information as the **JS Heap** line in the **Memory** chart:
+
+![Memory metrics](./reference-images/memory-chart.png)
+
+The colored lines on the chart map to the colored checkboxes above the chart.  Clear a checkbox to hide that category from the chart.
+
+The chart only displays the region of the recording that is currently selected.  For example, in the previous figure, the **Memory** chart is only showing memory usage from around the 3600 ms mark to the 6200 ms mark.
+
+See also:
+* [Visualize memory leaks (Performance tool: Memory checkbox)](#visualize-memory-leaks-performance-tool-memory-checkbox), below.
+
+
+<!-- ------------------------------ -->
+#### View the duration of a portion of a recording
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#duration -->
+
+When analyzing a section like **Network** or **Main**, sometimes you need a more precise estimate of how long certain events took.  Hold **Shift**, click and hold, and drag left or right to select a portion of the recording.  At the bottom of your selection, DevTools shows how long that portion took:
+
+![Viewing the duration of a portion of a recording](./reference-images/main-duration.png)
+
+
+<!-- ------------------------------ -->
+#### View a screenshot
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#view-screenshot -->
+
+To learn how to turn on screenshots, see [Capture screenshots while recording](#capture-screenshots-while-recording), above.
+
+To view a screenshot of how the page looked during that moment of the recording, hover over the **Overview**.  The **Overview** is the section that contains the **CPU** and **NET** charts:
+
+![Viewing a screenshot](./reference-images/screenshots-hover.png)
```

Comment:
This image is also broken in the preview.

<img width="630" alt="{CEA7C2D4-5372-44C6-B1B8-93D40F9E82E0}" src="https://github.com/user-attachments/assets/982c0a26-34f3-43c6-8808-2f5c5f68b6ae" />

Reply:
@captainbrosset  pls try:

1. go to internal preview anchor:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/performance/reference?branch=pr-en-us-3378#view-a-screenshot
2. In Edge, **F12** (DevTools) > Right-click **Refresh** > **Empty cache and hard refresh** > **F12**.

Predict the pngs are rendered.
 
Expect that to work, b/c the pngs are rendered at the below, and it is a frequent behavior/ action required.

external preview anchor:
https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/perf-sync/microsoft-edge/devtools-guide-chromium/performance/reference.md#view-a-screenshot